UnitsNet/UnitDefinitions/Irradiation.json

```diff
@@ -0,0 +1,33 @@
+﻿{
+  "Name": "Irradiation",
+  "BaseUnit": "JoulePerSquareMeter",
+  "XmlDoc": "Irradiation is the process by which an object is exposed to radiation. The exposure can originate from various sources, including natural sources.",
+  "XmlDocRemarks": "https://en.wikipedia.org/wiki/Irradiation",
+  "Units": [
+    {
+      "SingularName": "JoulePerSquareMeter",
+      "PluralName": "JoulesPerSquareMeter",
+      "FromUnitToBaseFunc": "x",
+      "FromBaseToUnitFunc": "x",
```

Comment:
Any prefix units to consider?
Any variants of area to consider; squarecentimeter etc ?

Reply:
I'm sorry, we only had occurrence of these specific units, we didn't have any other prefix/variants of that physical quantity in our domain knowledge.